grayskull/pypi/pypi.py

```diff
@@ -552,6 +552,12 @@ def _get_metadata(self) -> dict:
         version = ""
         if self["package"]["version"].values:
             version = self.get_var_content(self["package"]["version"].values[0])
+        if name.startswith("http://"):
+            print_msg(
+                f"Sorry. Grayskull presently only supports PyPI packages."
+            )
+            sys.exit()
```

Comment:
Are there other instances of using `sys.exit()` in this codebase? It's usually not a good idea to exit from a library unless it's directly exposed to the CLI (raising an exception + catching it in the CLI layer would work better), but if other parts of the code are doing it too, then I guess it's ok.